sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventDirectoryPage.java

```diff
@@ -48,6 +48,9 @@
       By.cssSelector("[id='relevanceStatusFilter'] [class='v-filterselect-button']");
   public static final By FILTER_BY_DISEASE =
       By.cssSelector("[id='disease'] [class='v-filterselect-button']");
+  public static final By EVENT_DISEASE_COMBOBOX =
+      By.xpath(
```

Comment:
no.. this type of locator is not accepted. If a developer will change any div from that nest, we'll have to fix it again.